lib/core/src/model.rs

```diff
@@ -295,15 +295,37 @@ pub trait EventListener: MaybeSend + MaybeSync {
 /// to listen for emitted events.
 #[derive(Clone, Debug, PartialEq)]
 pub enum SdkEvent {
-    PaymentFailed { details: Payment },
-    PaymentPending { details: Payment },
-    PaymentRefundable { details: Payment },
-    PaymentRefunded { details: Payment },
-    PaymentRefundPending { details: Payment },
-    PaymentSucceeded { details: Payment },
-    PaymentWaitingConfirmation { details: Payment },
-    PaymentWaitingFeeAcceptance { details: Payment },
-    Synced,
+    PaymentFailed {
+        details: Payment,
+    },
+    PaymentPending {
+        details: Payment,
+    },
+    PaymentRefundable {
+        details: Payment,
+    },
+    PaymentRefunded {
+        details: Payment,
+    },
+    PaymentRefundPending {
+        details: Payment,
+    },
+    PaymentSucceeded {
+        details: Payment,
+    },
+    PaymentWaitingConfirmation {
+        details: Payment,
+    },
+    PaymentWaitingFeeAcceptance {
+        details: Payment,
+    },
+    /// Synced with mempool and onchain data
+    ChainSynced,
+    /// Synced with real-time data sync
+    RtDataSynced {
```

Comment:
I'd rather call it `RealTimeDataSynced` or simply `DataSynced` than have to explain what Rt means

Reply:
Renamed to `DataSynced` in [207b692](https://github.com/breez/breez-sdk-liquid/pull/800/commits/207b692ef79e3d4080f8eca4da94d12a24498144)